app/test/api/meldekortperioder/MeldekortPeriodeTest.kt

```diff
@@ -1,86 +1,49 @@
 package api.meldekortperioder
 
-import api.TestConfig
-import api.api
 import api.kelvin.MeldekortPerioderDTO
-import api.util.ArenaClient
-import api.util.AzureTokenGen
-import api.util.Fakes
-import api.util.PostgresTestBase
-import com.fasterxml.jackson.databind.SerializationFeature
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
-import io.ktor.client.*
+import api.util.TestBase
 import io.ktor.client.call.*
-import io.ktor.client.plugins.contentnegotiation.*
 import io.ktor.client.request.*
 import io.ktor.http.*
-import io.ktor.serialization.jackson.*
-import io.ktor.server.testing.*
+import kotlinx.coroutines.runBlocking
 import no.nav.aap.arenaoppslag.kontrakt.intern.InternVedtakRequest
-import no.nav.aap.komponenter.dbtest.InitTestDatabase
 import no.nav.aap.komponenter.type.Periode
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Test
 import java.time.LocalDate
 
-private val dataSource2 = InitTestDatabase.freshDatabase()
-
-class MeldekortPeriodeTest : PostgresTestBase(dataSource2) {
+class MeldekortPeriodeTest : TestBase() {
 
     @Test
     fun `kan lagre ned og hente meldekort perioder`() {
-        Fakes().use { fakes ->
-            val config = TestConfig.default(fakes)
-            val azure = AzureTokenGen("test", "test")
-
-            testApplication {
-                application {
-                    api(
-                        config = config,
-                        datasource = dataSource2,
-                        arenaRestClient = ArenaClient()
+        runBlocking {
+            val perioder = listOf(
+                Periode(LocalDate.ofYearDay(2021, 1), LocalDate.ofYearDay(2021, 15)),
+                Periode(LocalDate.ofYearDay(2021, 16), LocalDate.ofYearDay(2021, 31))
+            )
+
+            val res = httpClient.post("/api/insert/meldeperioder") {
+                bearerAuth(azure.generate(true))
+                contentType(ContentType.Application.Json)
+                setBody(
+                    MeldekortPerioderDTO(
+                        "12345678910",
+                        perioder
```

Comment:
```suggestion
  fun `kan lagre ned og hente meldekort perioder`() =     runBlocking {
...}
```

Litt færre krøllparentes da :)